lib/puppet/util/servicenow.rb

```diff
@@ -233,3 +235,11 @@ def calculate_report_message_key_hash(report_status, resource_statuses)
   end
   module_function :calculate_report_message_key_hash
 end
+
+def report_labels(resource_statuses)
+  event_conditions = calculate_event_conditions(resource_statuses).select { |_, present| present == true }
```

Comment:
Note that `calculate_event_conditions` will not include the "audit" condition even if it's present. Not sure if we want to exclude audit here. Might have to modify that method if we want to include it.

Reply:
Good catch. Maybe you can help me come up with strategy for addressing this tomorrow.

Reply:
It looks like we might be able to just add an audit key and logic to populate it in calculate_event_conditions. I don't think that will break anything else.

Reply:
So looking at `audit` status documentation, that would be pretty much every resource in a run that didn't have something happen. I think pretty much every run that actually ran resources, unless catalog compilation failed, would have this status. For a PE run, possibly hundreds of resources would have this status for every run.

I think that the audit status isn't really adding anything useful to what we would want to know about a run, so I'm fine just leaving it out.

Reply:
Documentation: https://puppet.com/docs/puppet/6.17/format_report.html
> audit: Property was in sync and was being audited. Inspect reports are always in audit status.

Reply:
sounds good then!